pkg/evpn/bridge.go

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (c) 2022-2023 Intel Corporation, or its subsidiaries.
+// Copyright (c) 2022-2023 Dell Inc, or its subsidiaries.
+
+// Package evpn is the main package of the application
+package evpn
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"log"
+	"net"
+	"path"
+
+	"github.com/vishvananda/netlink"
+
+	pb "github.com/opiproject/opi-api/network/evpn-gw/v1alpha1/gen/go"
+
+	"go.einride.tech/aip/fieldbehavior"
+	"go.einride.tech/aip/fieldmask"
+	"go.einride.tech/aip/resourceid"
+	"go.einride.tech/aip/resourcename"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/emptypb"
+)
+
+// CreateLogicalBridge executes the creation of the LogicalBridge
+func (s *Server) CreateLogicalBridge(_ context.Context, in *pb.CreateLogicalBridgeRequest) (*pb.LogicalBridge, error) {
```

Comment:
We need to through an error if vlan < 1 or vlan > 4095:

Reply:
added